osu.Game.Tests/Database/BeatmapImporterTests.cs

```diff
@@ -670,6 +670,116 @@ public void TestImportThenDeleteThenImportOptimisedPath()
             });
         }
 
+        [Test]
```

Comment:
I wanted to put these in a separate file, but splitting out this test feels like it should be done as a separate pass. There's a lot of helper methods and reorganisation to be done.